### ISSUE TITLE
Unchecky is blocking Adguard browser extension 

### ISSUE THREAD

**Issue by rbeznosko-sya** [AdguardTeam/AdguardBrowserExtension#784]
Unchecky is blocking Adguard browser extension 

![image](https://user-images.githubusercontent.com/29629815/28311959-110cc6bc-6bba-11e7-9df9-24f505b80586.png)

### Steps to reproduce

1. Download Unchecky program.
2. After some time download Adg extension 
3. In couple minutes you will receive pop-up as on screenshot above.

### Your environment

* Adguard AdBlocker extension version: Adguard AdBlocker  2.6.7 for Google Chrome
* Environment name and version: Version 59.0.3071.115 (Official Build) (64-bit)
* Filters you use in Adguard: English filter, Spyware filter, Social media filter, Filter unblocking useful ads, Russian filter, RU AdList, Malware Domains
* Any other browser extensions you have installed:

**Comment by ameshkov** [comment]
Duplicate of https://github.com/AdguardTeam/AdguardForWindows/issues/1586

The issue was reported to unchecky author and should be fixed eventually:
http://unchecky.userecho.com/topics/744-unchecky-messing-with-adguard-extensions/
